torch/csrc/jit/frontend/tracer.cpp

```diff
@@ -982,7 +982,8 @@ void ArgumentStash::stashIntArrayRefElem(
   // TODO: check type?
   if (!isTracing())
     return;
-  auto& list_trace = stash.intlists.emplace(arg_name, size).first->second;
+  IntArrayRefTrace& list_trace =
+      stash.intlists.emplace(arg_name, size).first->second;
```

Comment:
Why this change is needed?

Reply:
The type to list_trace wasn't very clear when I was reading the code on the arg stash. So I thought to type it out to increase readability